src/app/components/map/core-map/core-map.component.ts

```diff
@@ -1,19 +1,20 @@
 import {Component, OnInit} from '@angular/core';
 import {latLng, LatLng, Map, tileLayer} from 'leaflet';
 import {FirePolygonLayer} from '../layer/fire-polygon.layer';
-import {TimeService} from '../../../services/time/time.service';
-import {FireService} from '../../../services/fire/fire.service';
 import {FireTweetLayer} from '../layer/fire-tweet.layer';
 import {TweetService} from '../../../services/tweet/tweet.service';
+import {TimeService} from '../../../services/time/time.service';
+import {FireService} from '../../../services/fire/fire.service';
+import {MapService} from '../../../services/map/map.service';
 
 @Component({
   selector: 'app-core-map',
   templateUrl: './core-map.component.html',
   styleUrls: ['./core-map.component.css']
 })
 export class CoreMapComponent implements OnInit {
-  constructor(private timeService: TimeService, private fireService: FireService, private tweetService: TweetService) {
-
+  constructor(private mapService: MapService, private timeService: TimeService, private fireService: FireService, private tweetService: TweetService) {
+    this.mapService.zoomIn.subscribe(this.zoomInToPolygon);
```

Comment:
change the event emitter name to `zoomInToPolygonEvent`

Let's use `Event` suffix for event emitters